course_catalog/models.py

```diff
@@ -149,25 +146,26 @@ class Meta:
         abstract = True
 
 
-class LearningPath(List):
+class UserList(List):
     """
-    LearningPath is a user-created model tracking a restricted list of LearningResources.
+    UserList is a user-created model tracking a restricted list of LearningResources.
     """
 
     author = models.ForeignKey(User, on_delete=models.PROTECT)
     privacy_level = models.CharField(max_length=32, null=True, blank=True)
     image_src = models.ImageField(
-        null=True, max_length=2083, upload_to=learning_path_image_upload_uri
+        null=True, max_length=2083, upload_to=user_list_image_upload_uri
     )
+    list_type = models.CharField(max_length=128)
```

Comment:
This looks like it has a `default` value in the migration but not here, could you reconcile that?

Reply:
The migration file asked for a default because it has to assume there already exists instances that are missing values for this new column. In reality, there aren't any instances yet, and instead of adding a default to the model declaration making a explicit declaration optional, I think I would prefer the list_type to be a required field that is explicitly declared. Do you disagree?

Reply:
ah, ok, that makes sense